addons/analytic/models/analytic_plan.py

```diff
@@ -147,8 +147,10 @@ def _compute_analytic_account_count(self):
 
     @api.depends('account_ids', 'children_ids')
     def _compute_all_analytic_account_count(self):
+        plans_data = self.env['account.analytic.account'].read_group([('root_plan_id', 'in', self.ids)], ['id:count'], ['root_plan_id'])
+        mapped_data = {data['root_plan_id'][0]: data['root_plan_id_count'] for data in plans_data}
```

Comment:
Use `_read_group` to avoid these dict and tuple manipulations (and avoid some unnecessary computations)